src/index.js

```diff
@@ -20,24 +20,22 @@ type Options = {
   port?: number,
   unsecure?: boolean,
   certsDir?: string,
-  web3url?: string,
+  ethNetwork?: string,
 }
 
 const start = async (opts: Options) => {
   const httpUrl =
     opts.httpUrl || SWARM_HTTP_URL || 'https://onyx-storage.mainframe.com'
   const wsUrl = opts.wsUrl || SWARM_WS_URL || 'ws://localhost:8546'
   const certsDir = opts.certsDir || 'certs'
-  const web3Url =
-    opts.web3url || WEB3_URL || 'https://rinkeby.infura.io/36QrH5cKkbHihEoWH4zS'
```

Comment:
Ah you had it like this before. I would suggest revert

Reply:
Yes that's the way I was thinking about it as well, but I think we'll still have to keep a default URL using Infura and if not in this repository, it will be in the client one, as I don't think we should ask end-users to provide an URL themselves.

Reply:
Agreed